source/Octopus.Cli/Commands/Releases/ReleasePlanBuilder.cs

```diff
@@ -194,14 +198,24 @@ async Task<ReleasePlan> BuildReleaseFromDatabase(IOctopusAsyncRepository reposit
             return plan;
         }
 
-        static async Task<Dictionary<string, FeedResource>> LoadFeedsForSteps(IOctopusAsyncRepository repository, ProjectResource project, IEnumerable<ReleasePlanItem> steps)
+        async Task<Dictionary<string, FeedResource>> LoadFeedsForSteps(IOctopusAsyncRepository repository, ProjectResource project, IEnumerable<ReleasePlanItem> steps)
         {
             // PackageFeedId can be an id or a name
             var allRelevantFeedIdOrName = steps.Select(step => step.PackageFeedId).ToArray();
```

Comment:
Can we rename this to `allRelevantFeedIds`?

Reply:
Done!